src/config.js

```diff
@@ -88,4 +88,4 @@ const DEFAULT_CONFIG = {
 // needs to contain actual changes, not all default values - and these
 // stay up-to-date automatically.
 // There might not be a user-defined config, default to an empty object.
-export default deepmerge(DEFAULT_CONFIG, (window.owntracks || {}).config || {});
+export default deepmerge(DEFAULT_CONFIG, (window.owntracks || {}).config || {});
```

Comment:
This removes a newline at the end - please don't.

Reply:
@linusg I haven't made any changes here but it looks like it has been done :/